org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -110,6 +112,13 @@ import org.eclipse.xtext.validation.Issue
 		if (params.rootPath === null) {
```

Comment:
Do we have an issue to support null as `rootPath`? It is a valid argument, the server should operate without the root folder then.